src/commands/projects.ts

```diff
@@ -85,13 +85,15 @@ export const builder = (argv: yargs.Argv) => {
                 .description,
             type: 'string',
             array: true,
+            deprecated: "Deprecated. Use 'ip-allow' command",
```

Comment:
Deprecating `--ip-allow` and `--ip-primary-only` options from the `projects update` command for now. We can remove it later in future.